### PR TITLE
full audit list servers

### LINKED ISSUE #2076: Full audit: List cloud servers and cloud server information
- https://support.rackspace.com/how-to/list-cloud-servers-and-cloud-server-information/

- From issue #1982 

- Email Cloud Escalations team for SME review

### PR DESCRIPTION
**Pull request by stephamon**
fixes #2076

### CODE REVIEW

**Comment by stephamon** [comment]
Hey @kmsholcomb can you give this an edit for me?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e4f66486cb/how-to/).

**Comment by kmsholcomb** [comment]
Hi @stephamon. Edits are attached. I think that this article provides more  information than it needs to! I don't think we should provide that list of servers from the nova client command, simply because it will always need updating (and we are, after all, telling users how to access that list themselves). And, I don't think that we should provide so many "server creation" steps in the control panel procedure. See my edits and comments for detailed suggestions, and see the attached mockup file for an idea of what I think the article could look like pared down. Please let me know if you have any questions. Thanks!

[List Cloud Servers and Cloud Server information_editsKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1191356/List.Cloud.Servers.and.Cloud.Server.information_editsKH.docx)
[List Cloud Servers and Cloud Server information_editsKH.pdf](https://github.com/rackerlabs/rackspace-how-to/files/1191357/List.Cloud.Servers.and.Cloud.Server.information_editsKH.pdf)
[List Cloud Servers and Cloud Server information_mockupKH.docx](https://github.com/rackerlabs/rackspace-how-to/files/1191355/List.Cloud.Servers.and.Cloud.Server.information_mockupKH.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c63fda3f4e/how-to/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-482844cb2d/how-to/)

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-c5c20eee0c/how-to/). :bow:

File: content/cloud-servers/view-information-about-available-cloud-servers.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+type: article
+created_date: '2013-10-23'
+created_by: Rose Contreras
+last_modified_date: '2017-08-01'
+last_modified_by: Stephanie Fillmon
+product: Cloud Servers
+product_url: cloud-servers
+---
+
+In addition to viewing CPU, RAM, and disk space configuration information for available cloud servers, you can also view disk, I/O, and networking information. Use this information to help you decide which cloud server is right for you needs. You can view the available servers through either the command-line nova client or the Cloud Control Panel GUI.
```
Comment (catlook) [inline content/cloud-servers/view-information-about-available-cloud-servers.md]: ... right for your needs ...

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-6045479801/how-to/). :bow:

**Pull request merged**